modules/core/03-connection/types/msgs_test.go

```diff
@@ -10,19 +10,20 @@ import (
 	abci "github.com/tendermint/tendermint/abci/types"
 	dbm "github.com/tendermint/tm-db"
 
-	"github.com/cosmos/ibc-go/testing/simapp"
 	"github.com/cosmos/cosmos-sdk/store/iavl"
 	"github.com/cosmos/cosmos-sdk/store/rootmulti"
 	storetypes "github.com/cosmos/cosmos-sdk/store/types"
-	sdk "github.com/cosmos/cosmos-sdk/types"
 	clienttypes "github.com/cosmos/ibc-go/modules/core/02-client/types"
 	"github.com/cosmos/ibc-go/modules/core/03-connection/types"
 	commitmenttypes "github.com/cosmos/ibc-go/modules/core/23-commitment/types"
 	ibctmtypes "github.com/cosmos/ibc-go/modules/light-clients/07-tendermint/types"
 	ibctesting "github.com/cosmos/ibc-go/testing"
+	"github.com/cosmos/ibc-go/testing/simapp"
 )
 
 var (
+	signer = "cosmos1ckgw5d7jfj7wwxjzs9fdrdev9vc8dzcw3n2lht"
```

Comment:
Is there a way to test in the testing package that this new feature will help with switching out bech32 prefixes?

Reply:
Yes and no. We could switch out the configs bech32 prefix like how the [relayer does](https://github.com/cosmos/relayer/blob/f93962fa8cc75bceb4377e11b09f8a3ae60579eb/relayer/chain.go#L455), but this would just be testing `.String()` function. 

`.String()` relies on the bech32 prefix set in the config. It is up to the caller of `.String()` to ensure the config has the currently set bech32 prefix. This pr just removes ibc-go from calling `.String()` and enforces the caller to have control over when `.String()` is called, which will hopefully lead the caller to avoid bugs and/or debug the issue easier